README.md

```diff
@@ -6,189 +6,95 @@
 
 ## Image tags ##
 
 The images of this container are tagged with [semantic
-versions](https://semver.org) of the underlying example project that they
-containerize.  It is recommended that most users use a version tag (e.g.
-`:0.0.1`).
+versions](https://semver.org) of the [admiral](https://github.com/cisagov/admiral)
+Python library that they containerize.  It is recommended that most users
+use a version tag (e.g. `:0.0.1`).
 
 | Image:tag | Description |
 |-----------|-------------|
-|`cisagov/example:1.2.3`| An exact release version. |
-|`cisagov/example:1.2`| The most recent release matching the major and minor version numbers. |
-|`cisagov/example:1`| The most recent release matching the major version number. |
-|`cisagov/example:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
-|`cisagov/example:nightly` | A nightly build of the `develop` branch of this repository. |
-|`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
-
-See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
+|`cisagov/admiral-docker:1.2.3`| An exact release version. |
+|`cisagov/admiral-docker:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/admiral-docker:1`| The most recent release matching the major version number. |
+|`cisagov/admiral-docker:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
+|`cisagov/admiral-docker:nightly` | A nightly build of the `develop` branch of this repository. |
+|`cisagov/admiral-docker:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
+
+See the [tags tab](https://hub.docker.com/r/cisagov/admiral-docker/tags) on Docker
 Hub for a list of all the supported tags.
 
 ## Volumes ##
 
 | Mount point | Purpose        |
 |-------------|----------------|
-| `/var/log`  |  Log storage   |
+| `mongo-init.js`  |  Stores the initialization script for MongoDB   |
 
 ## Ports ##
 
 The following ports are exposed by this container:
 
 | Port | Purpose        |
 |------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| 5555 | Celery Flower |
+| 6379 | Redis |
+| 8081 | Redis Commander |
+| 8083 | Mongo Express |
 
 ## Environment variables ##
 
 ### Required ###
 
-There are no required environment variables.
-
-<!--
 | Name  | Purpose | Default |
 |-------|---------|---------|
-| `REQUIRED_VARIABLE` | Describe its purpose. | `null` |
--->
+| `ADMIRAL_CONFIG_FILE` | Celery configuration | `admiral.yml` |
+| `CISA_HOME` | Home folder | `/home/cisa` |
+| `CISA_GROUP` | Group identifier | `cisa` |
```

Comment:
Any variable that has a default value is optional, by definition.

Reply:
I edited [51c8336](https://github.com/cisagov/admiral-docker/pull/5/commits/51c83362baf155eb8f446af7ceb224215ac1559d) to put all variables under the _Optional_ heading.

Reply:
Looks like this was taken care of in https://github.com/cisagov/admiral-docker/pull/5/commits/9f8b4140998172360a3e7593ea6a70827217ebd1. 👍